e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/navigating-to-utilisation-reports.spec.js

```diff
@@ -11,7 +11,7 @@ import USERS from '../../../fixtures/users';
 import { NODE_TASKS } from '../../../../../e2e-fixtures';
 import { aliasSelector } from '../../../../../support/alias-selector';
 
-context('PDC_RECONCILE users can route to the payments page for a bank', () => {
+context('Route to the payments page for a bank', () => {
```

Comment:
Lots of inconsistencies with the tests in this directory, especially with this change - what can we do to align? This is what we have with this change

```ts
context('PDC_RECONCILE users can add a payment to a report', () => {
context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing payments`, () => {
context(`${PDC_TEAMS.PDC_RECONCILE} users can delete payments`, () => {
context(`${PDC_TEAMS.PDC_RECONCILE} users can edit payments`, () => {
context(`${PDC_TEAMS.PDC_RECONCILE} users can filter fee records by facility id`, () => {
context(`users can filter payment details by facility id and payment reference and currency`, () => {
context('PDC_RECONCILE users can search for reports by bank and year', () => {
context('PDC_RECONCILE users can generate keying data', () => {
context('Route to the payments page for a bank', () => {
context('PDC_RECONCILE users can reconcile fee records', () => {
context(`${PDC_TEAMS.PDC_RECONCILE} users can remove fees from payments`, () => {
```